CMakeLists.txt

```diff
@@ -780,10 +780,8 @@ if (UNIX)
   # now that the user must set non-default -mNN we don't nec. need these:
```

Comment:
I believe this comment is now stale and can be removed.

Reply:
Reminder to address this comment.

Reply:
Done.